numpy/doc/indexing.py

```diff
@@ -294,6 +294,13 @@
 np.array([[1,2]]) (shape (1,2)) that is broadcast with the index array
 to produce a resultant array of shape (3,2).
```

Comment:
Hmm, this sentence is wrong too - this isn't broadcasting at all, as `(3,)` and `(1, 2)` do not broadcast.

I wonder if we should instead explain this as `y[:,1:3][np.array([0,2,4]),:]`? @seberg, thoughts?

Reply:
Uhh, right. I guess saying that it is the same as slicing everything first, and then taking the whole slice in the second index would work.
A bit ugly, but at least it makes the notion clear that there is in fact *no* broadcasting, etc. going on and slices always use orthogonal/outer semantics (they work independently).

So yeah, I think that is the best solution I can think of. Saying that they work independently and thus be separated out like that.

Reply:
> I wonder if we should instead explain this as `y[:,1:3][np.array([0,2,4]),:]`?
good point! maybe it is obvious, but it would be nice to put it in two python statements to indicate slicing followed by the index array usage.

Reply:
To push this forward: based on the above suggestions, I suggested a change here : https://github.com/numpy/numpy/pull/15891#discussion_r404341840 . I wasn't able to suggest change on L293, so we can commit this one and remove the L293 in a following commit. Alternatively, the code can just be pushed to @paulrougieux repo, but that probably needs to be done by a maintainer.